### DISCUSSION TITLE
IPV6 and AdGuardHome

### DISCUSSION THREAD

**Discussion by TheCeet** [AdguardTeam/AdGuardHome#3325] [Q&A]
AdGuardHome is running very smooth more then 1 year on my Pi4.
It's possible from my ISP to get IPV6. But I can't get it working on my home network with AGH.
I'm using an Asus router with Merlin firmware.
Someone toturial?

**Comment by ameshkov** [comment]
I don't fully understand what's the problem. You don't need to host AdGuard Home on an IPv6 address for IPv6 to work, DNS is independant.
